examples/runSingleFrameTask.py

```diff
@@ -22,27 +22,27 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
+from __future__ import print_function
 import os
 import sys
 import numpy
 
-import eups
-
-import lsst.daf.base               as dafBase
-import lsst.afw.table              as afwTable
-import lsst.afw.image              as afwImage
-import lsst.afw.display.ds9        as ds9
-import lsst.meas.algorithms        as measAlg
+from lsst.utils import getProductDir
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.afw.display.ds9 as ds9
+import lsst.meas.algorithms as measAlg
 from lsst.meas.algorithms.detection import SourceDetectionTask
 from lsst.meas.base import SingleFrameMeasurementTask
 
+
 def loadData():
     """Prepare the data we need to run the example"""
 
     # Load sample input from disk
-    mypath = eups.productDir("afwdata")
+    mypath = getProductDir("afwdata")
     if not mypath:
-        print >> sys.stderr, "Please setup afwdata and try again"
+        print("Please setup afwdata and try again", file=sys.stderr)
         sys.exit(1)
```

Comment:
This code is incorrect anyhow. It should trigger a test skip if afwdata is missing. It should not simply exit (pytest will break).